.travis.yml

```diff
@@ -47,4 +70,13 @@ install:
     - curl-config --version; pip freeze
 
 script:
-    - "tox -e $TOX_ENV -- $TOX_ARGS"
+    - |
+      # Skipping the tests for which we have already tested using it's own built wheel 
+      if [[ -z ${BUILD_WHEEL} ]]; then
+         tox -e $TOX_ENV -- $TOX_ARGS
+      else
+         pip install -U cibuildwheel
+         cibuildwheel --output-dir dist
+         sudo rm -rf dist/*manylinux2010_x86_64.whl
```

Comment:
@ploxiln I got a notification for your comment in email but github's UI has eaten it so I'm starting a new thread here.

It looks like what's going on is that `cibuildwheel` is producing two wheel files, one tagged `manylinux1` and one tagged `manylinux2010`. Then, because `cibuildwheel` doesn't seem to produce a manifest of what it built and encourages you to just do `*.whl`, there's no clean way to handle the output. 

I don't know what's going on internally with `cibuildwheel` and `manylinux` versions, but for our simple wheel it seems unlikely that we need to produce multiple versions. Can we limit it to just one manylinux version?

Reply:
It looks like we could possibly set `CIBW_MANYLINUX_X86_64_IMAGE=manylinux2010` and/or `CIBW_MANYLINUX_AARCH64_IMAGE=manylinux2014`

https://cibuildwheel.readthedocs.io/en/stable/options/#manylinux-image

> For architectures other than x86 (x86_64 and i686) manylinux2014 must be used because this is the first version of the manylinux specification that supports additional architectures.

Reply:
@bdarnell, yes it is creating two wheel files with two different tags(manylinux1 and manylinux2010) in x86 machine and as @ploxiln suggested we can specify `CIBW_MANYLINUX_X86_64_IMAGE=manylinux1` for x86 platform to avoid above issue.
Please check and let me know if any more changes required.